blood/src/main/java/cz/cuni/mff/d3s/blood/node_origin_tracker/DependencyMatrixCollector.java

```diff
@@ -56,6 +57,11 @@
         phaseOrder.add(NodeTracker.DeletedPhaseDummy.class);
     }
 
+    public DependencyMatrixCollector() {
```

Comment:
Should the constructor not be private?

Reply:
Reading further, I see that this is not the only case. I will not comment.

Reply:
Yup. Thanks, will fix.